pages/index.tsx

```diff
@@ -476,3 +506,8 @@ export default function IndexPage() {
     </>
   );
 }
+
+IndexPage.getInitialProps = () => {
+  const pluginName = randomPlugin("destination");
```

Comment:
I dig the randomness here, but I think we want to control it a bit more. Sailthru doesn't have as big on an impact and future ones will be more more niche.

Suggestion:
```
export function homePagePlugin() {
  const names = [
    "Marketo",
    "Marketo",
    "Salesforce",
    "Salesforce",
    "Zendesk",
    "Mailchimp",
  ];
  const pluginName = names[Math.floor(Math.random() * names.length)];

  return pluginName;
}
```